designer/server/src/main/scala/pl/touk/nussknacker/ui/config/UsageStatisticsReportsConfig.scala

```diff
@@ -2,7 +2,7 @@ package pl.touk.nussknacker.ui.config
 
 final case class UsageStatisticsReportsConfig(
     enabled: Boolean,
-    errorReportsEnabled: Boolean,
+    errorReportsEnabled: Boolean = true,
```

Comment:
Why this is changed?

Reply:
Good catch. It should be changed in common-designer.conf instead